python/dbsp/connection.py

```diff
@@ -58,20 +66,36 @@ def create_or_replace_program(self, *, name: str, sql_code: str, description: st
             DBSPProgram
         """
 
-        return self.create_program_inner(name = name, sql_code = sql_code, description = description, replace = True)
+        return self.create_program_inner(
+            name=name, sql_code=sql_code, description=description
+        )
 
-    def create_program_inner(self, *, name: str, sql_code: str, description: str, replace: bool):
+    def create_program_inner(self, *, name: str, sql_code: str, description: str):
```

Comment:
Seems wrong to always replace existing program.  The `replace` flag may have been useful.  But this whole API will probably get refactored, so I'm ok with this.

Reply:
in pipeline and connector we also update if something with this name already exists otherwise its created